src/BloomBrowserUI/BloomBrowserUI.csproj

```diff
@@ -239,6 +247,11 @@
     <Content Include="bookEdit\html\font-awesome\fonts\fontawesome-webfont.woff" />
     <Content Include="bookEdit\StyleEditor\StyleEditor.less" />
     <Content Include="bookLayout\basePage.less" />
+    <Content Include="pageChooser\blank-preview.jade" />
+    <Content Include="pageChooser\js\page-chooser.js.map" />
```

Comment:
Is it helpful to check these in? If someone is working on this file their build process will regenerate it and I don't think it's needed except for debugging?